docs/source/user/index.rst

```diff
@@ -3,28 +3,76 @@
 
+.. note::
+
+    Much of the documentation here is legacy documentation from FAST v8. While most of it is still
+    directly applicable to OpenFAST, portions may be out of date.
+
+
+.. _general-reference-docs:
+
+General
+~~~~~~~
 .. toctree::
    :maxdepth: 1
 
+   fast_to_openfast.rst
    api_change.rst
    input_file_overview.rst
-   aerodyn/index.rst
-   aerodyn-olaf/index.rst
-   aerodyn-aeroacoustics/index.rst
-   beamdyn/index.rst
-   subdyn/index.rst
-   elastodyn/index.rst
-   hydrodyn/index.rst
-   inflowwind/index.rst
-   servodyn/index.rst
-   servodyn-stc/StC_index.rst
-   fast_to_openfast.rst
-   cppapi/index.rst
-   fast.farm/index.rst
 
+Workshop material, legacy documentation, and other resources are listed below.
+
+- `Workshop Presentations <https://drive.google.com/drive/folders/1BDDfcnIyvmZCwf7eFo0ISI7aF_FMAOvt?usp=sharing>`_
+- :download:`FAST 8 User’s Guide <../../OtherSupporting/FAST8_UsersGuide.pdf>`
+- :download:`FAST v8 README <../../OtherSupporting/FAST8_README.pdf>`
+- `Implementation of Substructure Flexibility and Member-Level Load Capabilities for Floating Offshore Wind Turbines in OpenFAST <https://www.nrel.gov/docs/fy20osti/76822.pdf>`_
+- `FAST modularization framework for wind turbine simulation: full-system linearization <https://www.nrel.gov/docs/fy17osti/67015.pdf>`_
+- `Full-System Linearization for Floating Offshore Wind Turbines in OpenFAST <https://www.nrel.gov/docs/fy19osti/71865.pdf>`_ 
+- :download:`FAST with Labview <../../OtherSupporting/UsingFAST4Labview.pdf>`
+
+
+Module Documentation
+~~~~~~~~~~~~~~~~~~~~
+This section contains documentation for the OpenFAST module-coupling environment and its underlying modules.
+Documentation covers usage of models, underlying theory, and in some cases module verification.
+
+.. toctree::
+   :maxdepth: 1
+
+   AeroDyn <aerodyn/index.rst>
+   OLAF <aerodyn-olaf/index.rst>
+   Aeroacoustics <aerodyn-aeroacoustics/index.rst>
+   BeamDyn <beamdyn/index.rst>
+   SubDyn <subdyn/index.rst>
+   ElastoDyn <elastodyn/index.rst>
+   HydroDyn <hydrodyn/index.rst>
+   InflowWind <inflowwind/index.rst>
+   ServoDyn <servodyn/index.rst>
+   Structural Control <servodyn-stc/StC_index.rst>
+   C++ API <cppapi/index.rst>
+   FAST.Farm <fast.farm/index.rst>
+   
+
+Modularization Framework
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Information specific to the modularization framework of OpenFAST is provided here. These are a collection
+of publications, presentations, and past studies on the subject.
+
+- `The New Modularization Framework for the FAST Wind Turbine CAE Tool <https://www.nrel.gov/docs/fy13osti/57228.pdf>`_
+- :download:`New Module Development Plan <../../OtherSupporting/ModulePlan_GasmiPaperExamples.doc>`
+- :download:`Linearization of Mesh Mapping <../../OtherSupporting/LinearizationOfMeshMapping_Rev18_Rev2.doc>`
+- :download:`Interpolation of DCMs <../../OtherSupporting/DCM_Interpolation.pdf>`
```

Comment:
This document is actually already included in the "5.4 Other Documentation" section of readthedocs.  Suggesting deleting it from there and only including it here.